src/routes/index.ts

```diff
@@ -298,6 +298,12 @@ router.post(config.MANAGING_OFFICER_CORPORATE_URL, authentication, navigation.ha
 router.post(config.MANAGING_OFFICER_CORPORATE_URL + config.ID, authentication, navigation.hasBeneficialOwnersStatement, ...validator.managingOfficerCorporate, checkValidations, managingOfficerCorporate.update);
 router.get(config.MANAGING_OFFICER_CORPORATE_URL + config.REMOVE + config.ID, authentication, navigation.hasBeneficialOwnersStatement, managingOfficerCorporate.remove);
 
+router.get(config.MANAGING_OFFICER_CORPORATE_WITH_PARAMS_URL, authentication, navigation.hasBeneficialOwnersStatement, managingOfficerCorporate.get);
```

Comment:
For the other PRs we've done, we've grouped each new 'with params' route with its non 'with params' counterpart, rather than grouping them all together like this. I think we should be consistent and follow the same layout.

Reply:
I've done it like this in my PR. Either way, it's pretty obvious what needs to be deleted when we come to remove the flag.

Reply:
Grouped